benchmarks/Other/Tokenizers/KSkipNGramBench.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rubix\ML\Benchmarks\NeuralNet\ActivationFunctions;
```

Comment:
This should be `namespace Rubix\ML\Benchmarks\Others\Tokenizers;` I believe

Reply:
Yes, thank you, I've just copy pasted other benchmark file from the same folder, fixed.
But it gave me the idea to check all other files from the same directory.